ui/src/app/streams/streams.service.ts

```diff
@@ -49,8 +52,21 @@ export class StreamsService {
       this.streamDefinitions.pageNumber,
       this.streamDefinitions.pageSize
     );
-      // TODO Implement Sorting
-      // params.sort = pageable.calculateSortParameter();
+
+    if (definitionSort != null) {
```

Comment:
undefined instead of null.  Will fix on merge.

Reply:
Ah, we didn't change these in TasksService and as model was same I just copy/pasted. Weird as I remember some nulls were changed to undefined with that older PR.

Reply:
Updating the task service as a part of the polishing on this PR.